src/redux-flow/reducers/tic-tac-toe/index.js

```diff
@@ -2,12 +2,14 @@ import {
   FILL_SQUARE,
   RESET_GAME,
   JOIN,
+  JOIN_ROOM,
```

Comment:
I don't like the senames... what's the difference between join and join room?

To figure this out one must read almost the entire code. Isn't there a better name?